projects/spirals-chaos/index.md

```diff
@@ -0,0 +1,126 @@
+--- 
+authors: 
+  - "Paulina Mrksic"
+authors-url: 
+  - "https://github.com/paulinalinaa"
+layout: project
+title: "spirals chaos"
+type: project
+---
+
+![](splash.png)￼
```

Comment:
relative paths should be 

    ![](./splash.png)


Also later on the images should be:  

    ![](./assets/images/image.png)